tests/test_views_login.py

```diff
@@ -7,8 +7,8 @@
 from django.urls import reverse
 from django_otp import DEVICE_ID_SESSION_KEY
 from django_otp.oath import totp
-from django_otp.util import random_hex
 
+from two_factor.models import random_hex_str
 from .utils import UserMixin
```

Comment:
Just one space blank line between lines 11 and 12 and isort will be happy :smiley_cat: